README.md

```diff
@@ -1,6 +1,20 @@
 Sitemap (optional module)
 =========================
 
+## Migration
+
+On Jahia >= 8.0.3 migration should happen automatically provided that you have previous version of the module installed.
```

Comment:
Automatically only if the groovy script is added in the Jahia patches folder

Reply:
It is in the patches folder by default

Reply:
I meant in the Jahia patches folder. sitemap is not provided with the Jahia distribution where the patches are now executed at startup. But if the user add the new version in the migrated Jahia, will the patch provided by the module be executed at startup like the others ?

Reply:
Yes, the patches in the module are independent of what we add in Jahia so they will run on first startup of this version on 8.0.3 Jahia.